tlv/internal/gen/gen_tlv_types.go

```diff
@@ -12,8 +12,13 @@ const (
 
 	// customTypeStart defines the beginning of the custom TLV type range as
 	// defined in BOLT-01.
-	customTypeStart   uint32 = 65536
-	defaultOutputFile        = "tlv_types_generated.go"
+	customTypeStart uint32 = 65536
+
+	// gossipUnsignedTypeStart defines the beginning of TLV range used in
+	// Gossip 1.75 messages that will not be covered by signatures.
+	gossipUnsignedTypeStart uint32 = 160
```

Comment:
Should we instead just increase `numberOfTypes` to 300 or something to that effect?

Reply:
ja could do! also need to add 2 marker types (one for 1000000000 and one for 3000000000). 

will update soon

Reply:
ah - realised i dont need the markers. So just did your suggestion and bumped the num gen to 300 :)